src/xbase/schematic/netlist_info/clamp_static.symbol.yaml

```diff
@@ -0,0 +1,175 @@
+lib_name: xbase
+cell_name: clamp_static
+view_name: symbol
+bbox:
```

Comment:
Did you ensure that the lower left coordinate of the red box in the symbol is at (0, 0), and all pins are on grid?

Reply:
Looks good to me. I did push one more update to draw a more useful symbol.